src/lib/classifyquote.tsx

```diff
@@ -39,7 +70,7 @@ melanie
 [5:23 PM]
 I just got home, so I'm gonna read them now!
 */
-const slackHeaderRegex = /^(.+?[^\s])( |$\n?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
+const slackVerboseHeaderRegex = /^(.+?[^\s])( |$(\r\n|\n)?)\[\d{1,2}:\d{2} (A|P)M\](\s+)?/;
```

Comment:
Also I encountered an issue when testing the fancy display frontend thing yesterday. Things like
```
Tiffany [5:15 PM]
shucks

[5:15 PM]
shucks
```

Were turning the second timestamp into part of the message